Organization profile first draft

LalithVipulananthan opened the pull request:
A first pass at an organization profile for the AWF

Commit:
Organization profile first draft

A first pass at an organization profile for the AWF

kenji-miyake commented:
@BonoloAWF cc @xmfcx @mitsudome-r What do you think about the content?

Commit:
Added brief description of each repository

Brief description of each repository was taken from an older version of the autoware repository's README, with some additional minor editing.

Signed-off-by: Lalith Vipulananthan <lalith.vipulananthan@tier4.jp>

xmfcx commented on profile/README.md:
```diff
@@ -1,12 +1,39 @@
-## Hi there 👋
+## Welcome to the Autoware Foundation! 👋
+![Autoware stack](https://static.wixstatic.com/media/984e93_552e338be28543c7949717053cc3f11f~mv2.png/v1/crop/x_0,y_1,w_1500,h_879/fill/w_1079,h_633,al_c,usm_0.66_1.00_0.01,enc_auto/Autoware-GFX_edited.png)
```
```suggestion
![Autoware stack](https://static.wixstatic.com/media/984e93_552e338be28543c7949717053cc3f11f~mv2.png/v1/crop/x_0,y_1,w_1500,h_879/fill/w_1079,h_633,al_c,usm_0.66_1.00_0.01,enc_auto/Autoware-GFX_edited.png)
```
Wow this image needs to be updated, it looks so outdated in style. I will create and send one today.

xmfcx replied:
Under the https://www.autoware.org/media-kit , there is the [Brand Guide pdf](https://www.autoware.org/_files/ugd/a700a5_31b49b4418564de1a91b61afa60e8462.pdf) and it depicts the colors allowed for the `.IO`, `.AI`, `.AUTO` projects.

Since it was so formal, I didn't want to make any banners for now.

@mitsudome-r @BonoloAWF I think we should bring discussion of branding of `.CORE` and `.UNIVERSE` to this night's TSC meeting, what do you think?

We don't even have the colors for these 2.

xmfcx replied:
I've created an issue to track this: https://github.com/autowarefoundation/.github/issues/2

xmfcx commented on profile/README.md:
```diff
@@ -1,12 +1,39 @@
+  - Core repository containing high-quality and stable ROS packages for Autonomous Driving.
+  - Based on [Autoware.Auto](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto) and [Autoware.Universe](https://github.com/autowarefoundation/autoware.universe).
+- [autowarefoundation/autoware.universe](https://github.com/autowarefoundation/autoware.universe)
+  - Core repository containing experimental but cutting-edge ROS packages for Autonomous Driving.
```
```suggestion
  - The main repository for experimental but cutting-edge ROS packages for Autonomous Driving.
```

xmfcx replied:
Sure, it sounds reasonable!

xmfcx commented on profile/README.md:
```diff
@@ -1,12 +1,39 @@
+  - Library/utility type repository containing commonly referenced ROS packages. 
+  - These packages were moved to a separate repository in order to reduce CI execution time
+- [autowarefoundation/autoware.core](https://github.com/autowarefoundation/autoware.core)
+  - Core repository containing high-quality and stable ROS packages for Autonomous Driving.
```
```suggestion
  - The main repository for high-quality and stable ROS packages for Autonomous Driving.
```

xmfcx replied:
❤, I support your update on this change.

xmfcx commented on profile/README.md:
```diff
@@ -1,12 +1,39 @@
+  - Launch configuration repository containing node configurations and their parameters.
+- [autowarefoundation/autoware-github-actions](https://github.com/autowarefoundation/autoware-github-actions)
+  - Contains [reusable GitHub Actions workflows](https://docs.github.com/ja/actions/learn-github-actions/reusing-workflows).
+  - Since Autoware has many repositories, making CI scripts DRY is efficient.
```
```suggestion
  - Used by many repositories of the Autoware for CI.
  - Utilizes [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) concept.
```

xmfcx commented on profile/README.md:
```diff
@@ -1,12 +1,39 @@
+  - Since Autoware has many repositories, making CI scripts DRY is efficient.
+- [autowarefoundation/autoware-documentation](https://github.com/autowarefoundation/autoware-documentation)
+  - Documentation repository for Autoware users and developers.
+  - Since Autoware Core/Universe has multiple repositories, a central documentation repository is more user-friendly than writing distributed documentation in each repository.
```
```suggestion
  - Since Autoware Core/Universe has multiple repositories, a central documentation repository is important to make information accessible from a single place.
```

Each package will have its own documentation too so I made this change @kenji-miyake

kenji-miyake replied:
Great! Thank you. :pray:

xmfcx commented on profile/README.md:
```diff
@@ -1,12 +1,39 @@
-🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
--->
+### Contribution guidelines
+[There is no formal process to becoming a contributor!](https://github.com/autowarefoundation/autoware-projects/wiki#contributors). If you want to become a contributor then you can comment on any [existing issues](https://github.com/autowarefoundation/autoware.universe/issues) or make a pull request on any Autoware repository. 
```
```suggestion
[There is no formal process to become a contributor!](https://github.com/autowarefoundation/autoware-projects/wiki#contributors). If you want to become a contributor then you can comment on any [existing issues](https://github.com/autowarefoundation/autoware.universe/issues) or make a pull request on any Autoware repository. 

Make sure to follow the [Contribution Guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/)!
```

Maybe we can add the information in https://github.com/autowarefoundation/autoware-projects/wiki#contributors into the https://autowarefoundation.github.io/autoware-documentation/main/contributing/#contribution-workflows and only have a single source of information.

What do you think @kenji-miyake @LalithVipulananthan ?

kenji-miyake replied:
I agree with @xmfcx. :+1: 

[Nits] By the way, `the` is better than `our`.
https://github.com/autowarefoundation/autoware-documentation/issues/51

BonoloAWF commented:
Can you add a screenshot showing Autoware running in simulation at least? These should be readily available from past ODD demonstrations.

ghost commented:
> Can you add a screenshot showing Autoware running in simulation at least? These should be readily available from past ODD demonstrations.

Sure. Would it also be an idea to list the currently supported ODDs (autonomous valet parking, cargo delivery) as well as the plan to support the bus ODD?

kenji-miyake commented:
@LalithVipulananthan @BonoloAWF @xmfcx cc @yukkysaito Where do you think we should put this kind of image?
https://github.com/autowarefoundation/autoware/pull/104

xmfcx commented:
> @LalithVipulananthan @BonoloAWF @xmfcx cc @yukkysaito Where do you think we should put this kind of image? [autowarefoundation/autoware#104](https://github.com/autowarefoundation/autoware/pull/104)

@LalithVipulananthan could you put following image they are mentioning to top until we create a better one? It is the most promising picture I think. @BonoloAWF do you agree?

![image](https://user-images.githubusercontent.com/10751153/158829692-e741e20f-912d-4788-b1ce-625c73f3a2cf.png)

xmfcx committed:
Apply suggestions from code review

Co-authored-by: Lalith Vipulananthan <63835446+LalithVipulananthan@users.noreply.github.com>

Commit:
Apply suggestions from code review

Batch applying suggestions

Co-authored-by: M. Fatih Cırıt <xmfcx@users.noreply.github.com>

Commit:
Updated top image

Changelist:
- Added a link to the Autoware Foundation homepage at the top where the AWF is first mentioned
- Minor formatting changes around the Contribution Guidelines section
- Moved the Autoware Foundation link to the Useful resources section and removed the Appendix section as it seemed redundant with just the one link there

Commit:
Updated repository overview

xmfcx commented on profile/README.md:
```diff
@@ -1,12 +1,36 @@
 
-<!--
+The [Autoware Foundation](https://www.autoware.org/) manages the development of Autoware, the world's leading open-source software project for autonomous driving, and sponsors Autoware-based projects all over the world. Autoware was started in 2015 and is being used by over 100 companies on more than 30 vehicles in over 20 different countries worldwide. 
 
```
```suggestion

[![Discord](https://img.shields.io/discord/953808765935816715?label=Join%20Autoware%20Discord&style=for-the-badge)](https://discord.gg/Q94UsPvReQ)

```

xmfcx replied:
Added this here with suggestion made in https://github.com/autowarefoundation/autoware-documentation/pull/58#pullrequestreview-913859438

Commit:
Add Discord invite badge

Co-authored-by: M. Fatih Cırıt <xmfcx@users.noreply.github.com>

kenji-miyake reviewed (approved):
LGTM, but I think I have no right to make the decision.

As an improvement, we can consider introducing pre-commit (Prettier and markdownlint) in the same way as https://github.com/autowarefoundation/autoware.

xmfcx commented:
> LGTM, but I think I have no right to make the decision.

You do, as with any other member!

> As an improvement, we can consider introducing pre-commit (Prettier and markdownlint) in the same way as https://github.com/autowarefoundation/autoware.

Yeah, we should add it to this repo too. I just hastily created the repo with default settings.

kenji-miyake commented:
> Yeah, we should add it to this repo too. I just hastily created the repo with default settings.

Okay, I'll send a PR!
